.github/workflows/validate.yml

```diff
@@ -0,0 +1,33 @@
+name: ValidateTrialJsonSchema
+on:
+  push:
+    branches: [ main, feature/json-validation ]
+  pull_request:
+    branches: [ main, feature/json-validation ]
+
+
+jobs:
+  Validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Check changed files
+        id: calc_files_diff
+        run: |
+          git fetch origin main:localmain
+
+          # Gets changed json files excluding the json schema
+          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
+
+          echo "Files changed since last commit..."
+          echo "$FILES"
+
+          echo "::set-output name=diff::$( echo "$FILES" | tr '\n' ',' | sed '$s/,$/\n/' )"
+
+      - name: Validate JSON
+        if: ${{ steps.calc_files_diff.outputs.diff != '' }}
```

Comment:
While working on the PR notes, I identified a bug that was happening - when there were no files that had changed, the workflow would fail. This If statement fixes this bug.